packages/filesystem/src/browser/file-tree/file-tree-widget.tsx

```diff
@@ -152,14 +162,16 @@ export class FileTreeWidget extends TreeWidget {
             }
         }
     }
-
-    protected setTreeNodeAsData(data: DataTransfer, node: TreeNode): void {
-        data.setData('tree-node', node.id);
+    protected setTreeNodesAsData(data: DataTransfer, nodes: TreeNode[]): void {
```

Comment:
let's introduce new methods to set/get `nodes` instead of `node` for backward compatibility